python/utils.py

```diff
@@ -411,12 +411,23 @@ def _unloadPluginModules(packageName):
             if hasattr(sys.modules[mod], 'qCleanupResources'):
                 sys.modules[mod].qCleanupResources()
         except:
-            pass
+            # Print stack trace for debug
+            qDebug("qCleanupResources error:\n%s" % traceback.format_exc())
```

Comment:
I'm not sure this should be backported. It will probably mostly generate debug noise and not considerably improve the quality of the LTR

```suggestion
            qDebug("qCleanupResources error:\n%s" % traceback.format_exc())
```